### PR TITLE
Add datawrapper block and column width for iframes

### LINKED ISSUE #1370: Allow iframes to be wider than the column
@kevinhowbrook, I'm starting a new issue for this, per our chat! Since we'll often embed maps and other more intricate iframes/dataviz, can we allow them to be wider than the standard width of a column? The default settings also right-align iframes, so if we allow them to be wider, the iframe should then also be centered on the page.

Would love to have this done by May 1, since one of the projects we're launching on that date has a world map that would be well-served by being larger. But if it takes a few extra days beyond that, that's OK. (Would also be curious what it would take to allow iframes to be full-width, if possible? But that's definitely more of a nice-to-have rather than something we need in the immediate term.)

### PR DESCRIPTION
**Pull request by nmorduch**
Resolves #1370 and #1364

### CODE REVIEW

File: newamericadotorg/templates/base.html [hunk 1]
```diff
@@ -136,5 +136,9 @@
   </script>
   <!-- End Twitter universal website tag code -->
 
+  {# Datawrapper responsive embed js #}
+  <script type="text/javascript">!function(){"use strict";window.addEventListener("message",function(a){if(void 0!==a.data["datawrapper-height"])for(var e in a.data["datawrapper-height"]){var t=document.getElementById("datawrapper-chart-"+e)||document.querySelector("iframe[src*='"+e+"']");t&&(t.style.height=a.data["datawrapper-height"][e]+"px")}})}();</script>
```
Comment (nmorduch) [inline newamericadotorg/templates/base.html]: Cool cool cool looks like this isn't ending up on the page, so I'll go figure out why.

**Comment by nmorduch** [comment]
Hey @kevinhowbrook I think this one is finally good enough for production! All that needs review is this new commit https://github.com/newamericafoundation/newamerica-cms/pull/1384/commits/c21a783f2e23dd16a3c368502e019cf18073fd5f which has the only change (moving where the JS lives)

On staging, same links as before. Still some layout bugs if you do a wide one on pages other than reports, but we're just telling people not to use the wide options unless they're directed to.

- https://staging.newamerica.org/education-policy/reports/undermining-pell-iv/introduction/
- https://staging.newamerica.org/education-policy/reports/undermining-pell-iv/print/
- https://staging.newamerica.org/public-interest-technology/blog/ai-marketing-trojan-horse/
- https://staging.newamerica.org/bretton-woods-ii/about/

**Comment by kevinhowbrook** [comment]
> Still some layout bugs if you do a wide one on pages other than reports, but we're just telling people not to use the wide options unless they're directed to.
Are we planning on addressing this in a later issue?

**Comment by nmorduch** [comment]
Yeah, a very low priority much later issue :)

**Pull request merged**